Makefile

```diff
@@ -1,6 +1,21 @@
+GOPROXY ?= https://goproxy.io
```

Comment:
On both the `Dockerfile` and the `Makefile`: I think it's better to make "no proxy" as the default behavior (I.E., the default for global users, instead of for users who are behind the GFW without a VPN). The reason behind this is, we want to make it convenient for _the majority_ of the developers.

For developers behind the GFW without a VPN, they probably already have GOPROXY ENV var set in their local environment, so they can just run `make build-with-docker` without any change or extra parameter.

Reply:
It's not very accurate, the GOPROXY ENV is used inside the container, the env set in their local environment is not work.

If we make "no proxy" as the default behavior, many developers will get failed, we can consider it more.